Sming/Arch/Esp8266/Components/axtls-8266/axtls-8266.patch

```diff
@@ -21,8 +21,7 @@ diff -Nuar a/replacements/libc.c b/replacements/libc.c
 + Modified 03 April 2015 by Markus Sattler
 + */
 +
-+#include <stdint.h>
-+#include "espinc/c_types_compatible.h"
++#include <c_types.h>
 +#include <stdarg.h>
 +
 +extern int ets_putc(int);
```

Comment:
In this PR  `make dist-clean` did not clean the axtls directory and the new patches was not applied cleanly. Any idea why is that happening?

Reply:
Try `make dist-clean ENABLE_SSL=1`

Reply:
We have these optional custom libraries which get built:

	pwm_open
	mainmm
	lwip_full
	lwip_open
	lwip2
	axtls
	mqttc

Behaviour is that the library only gets cleaned if the relevant flags are defined. It would be better of course if they all got cleaned regardless. I can look at this in another PR.

Reply:
>  It would be better of course if they all got cleaned regardless. I can look at this in another PR.

Absolutely. It will also match the current default behaviour.